genai-perf/genai_perf/inputs/retrievers/payload_input_retriever.py

```diff
@@ -127,27 +127,49 @@ def _get_content_from_input_file(
                         )
                     prompt = prompt if prompt else prompt_alt
                     prompts.append(prompt.strip() if prompt else prompt)
-                    timestamp = data.get("timestamp", "")
+                    timestamp = self._get_valid_timestamp(data)
                     timestamps.append(timestamp)
                     optional_data = self._check_for_optional_data(data)
                     optional_datas.append(optional_data)
         return prompts, timestamps, optional_datas
 
+    def _get_valid_timestamp(self, data: Dict[str, Any]) -> int:
+        """
+        Retrieves and validates timestamp from input data
+        """
+        timestamp = data.get("timestamp")
+        if timestamp is None:
+            raise ValueError("Each data entry must have a 'timestamp' field.")
+        try:
+            timestamp = int(timestamp)
+        except ValueError:
+            raise ValueError("Invalid timestamp value.")
```

Comment:
I think we should give user more information on why it failed. Maybe like this:
```suggestion
raise ValueError(f"Invalid timestamp: Expecting an integer but received '{timestamp}'.")
```

Reply:
Will modify